Template.php

```diff
@@ -1371,7 +1371,7 @@ protected function use_unnamed_params() {
       }
       
       if (preg_match('~^(https?://|www\.)\S+~', $dat, $match)) { # Takes priority over more tenative matches
-        $this->set('url', $match[0]);
+        $this->add_if_new('url', $match[0]);
```

Comment:
If 'url' is already set, add_if_new should return `FALSE`; in this case, the unused URL might be assigned to a "duplicate_url" parameter?